.travis.yml

```diff
@@ -140,6 +158,11 @@ jobs:
       env: DB=mariadb MARIADB_VERSION=10.1
       addons:
         mariadb: 10.1
+    - stage: Test
+      php: 7.3
+      env: DB=mariadb MARIADB_VERSION=10.1
```

Comment:
Given MariaDB 10.0 is tested on PHP 7.3 and MariaDB 10.1 is tested on other PHP versions, it doesn't look like MariaDB 10.1 on PHP 7.3 provides any additional value. Let's try reducing the number of newly introduced jobs to the necessary minimum (e.g. only the newest version of each platform).